book/templates.rst

```diff
@@ -942,23 +942,14 @@ node and the ``ref`` attribute:
     <block name="blocks" default-type="text_block" minOccurs="0">
         <types>
             <type ref="text_block" />
-            <type name="editor">
-                <meta>
-                    <title lang="en">Editor</title>
-                    <title lang="de">Editor</title>
-                </meta>
-                <properties>
-                    <property name="text_editor" type="text_editor">
-                        <meta>
-                            <title lang="en">Text Editor</title>
-                            <title lang="de">Texteditor</title>
-                        </meta>
-                    </property>
-                </properties>
-            </type>
         </types>
     </block>
 
+.. note::
+
+    Mixing global and local block types is supported but it is recommended to keep your block type names unique,
+    to avoid confusion and change to global blocks in future easier without data migration.
```

Comment:
```suggestion
    Mixing global and local block types is supported, but it is recommended to keep your block type names unique
    to avoid confusion. This approach also simplifies the transition to global blocks in the future, eliminating the need
    for data migrations.
```